src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length">
+      <div class="calendar__day calendar__day--start-day_sun"></div>
```

Comment:
start-day should be a modifier for calendar (not for calendar__day)
```suggestion
    <div class="calendar calendar--month-length_28 calendar__day--start-day_sun">
      <div class="calendar__day"></div>
```